manifests/xgboost-operator/base/deployment.yaml

```diff
@@ -1,8 +1,14 @@
-apiVersion: extensions/v1beta1
+apiVersion: extensions/v1
```

Comment:
why v1 ?
https://github.com/kubeflow/mpi-operator/blob/master/deploy/v1/mpi-operator.yaml

do we follow this way?

Reply:
do you test this on the local machine ?

Reply:
Yes I tested on my local machine. v1beta1 doesn't work but v1 works,  and I also check the config file on mpi operator, they use v1 and works fine

Reply:
ok.